test/runtests.jl

```diff
@@ -98,7 +98,7 @@ include("util.jl")
         )
         @test chain1b isa Chains
         @test range(chain1b) == range(26; step=4, length=10_000)
-        @test mean(chain1b["μ"]) ≈ 0.0 atol=0.1
+        @test mean(chain1b["μ"]) ≈ 0.0 atol=0.15
```

Comment:
This change fixes some test failure, which is probably caused by the recent RNG changes in other packages such as Distributions. I saw the test error in AbstractMCMC earlier today, so I'm quite certain it is not caused by this PR.

Reply:
The same fix was also needed in #81.